tests/tests/certificates.spec.js

```diff
@@ -0,0 +1,97 @@
+/// <reference types="cypress" />
+import 'cypress-file-upload';
+
+const CERT_NAME = 'cypress-test-name';
+const CERT_COMMON_NAME = 'cypress-test-common-name';
+const ANNOTATION_KEY = 'annotation';
+const ANNOTATION_VALUE = 'value';
+
+context('Add and remove a certificate', () => {
+  before(() => {
+    cy.loginAndSelectCluster();
+    cy.goToNamespaceDetails();
+  });
+
+  it('Adds and displays a certificate', () => {
+    cy.getLeftNav()
+      .contains('Configuration')
+      .click();
+
+    cy.getLeftNav()
+      .contains('Certificates')
+      .click();
+
+    cy.url().should('match', /certificates$/);
+
+    cy.getIframeBody()
+      .contains('Create Certificate')
+      .click()
+      .should('be.visible');
+
+    cy.getIframeBody()
+      .contains('Simple')
+      .should('be.visible');
+
+    cy.getIframeBody()
+      .find('input[data-cy="cert-name"]:visible')
+      .type(CERT_NAME);
+
+    cy.getIframeBody()
+      .find('input[placeholder^="Certificate CN"]:visible')
```

Comment:
Damn, so we can instead
```
.find('yadadada')
.filter(':visible', { log: false })
```
do the simpler
```
.find('yadadada:visible')
```
?

Nice, we'll probably batch replace them in our tests after we finish writing all of them.